### PR TITLE
Wcp 31689 :- Updating the Documentation with the detailed process of cloning or downloading the script

### PR DESCRIPTION
**Pull request by anandsirasagi**

### CODE REVIEW

File: OracleWebCenterPortal/dockerfiles/12.2.1.4/README.md [hunk 1]
```diff
@@ -71,7 +71,8 @@ MANAGED_SERVER_PORT=<Managed Server Port>
 CONFIGURE_UCM_CONNECTION=<true or false>
 #Valid option for socket type are socket,jaxws 
 UCM_SOCKET_TYPE=<UCM Socket Type >
-#Required if UCM_SOCKET_TYPE is jaxws
+#If UCM_SOCKET_TYPE is socket configure UCM_HOST=(hostname or IPAddress) of UCM machine
+#If UCM_SOCKET_TYPE is jaxws configure UCM_HOST=(protocol://hostname or IPAddress) of UCM machine
 UCM_HOST=<UCM Host>
```
Comment (dgothe) [inline OracleWebCenterPortal/dockerfiles/12.2.1.4/README.md:76]: remove above two comments and instead use the one below
# Set to true if UCM is using SSL
UCM_USING_SSL=false

File: OracleWebCenterPortal/dockerfiles/12.2.1.4/container-scripts/createContentServerConnection.py [hunk 2]
```diff
@@ -40,7 +40,7 @@
 print('')
 print('')
 
-ucmUrl= "http://" +  ucmHost + ":" + ucmPort + "/" + "idcnativews"
+ucmUrl= ucmHost + ":" + ucmPort + "/" + "idcnativews"
```
Comment (dgothe) [inline OracleWebCenterPortal/dockerfiles/12.2.1.4/container-scripts/createContentServerConnection.py:43]: update based on UCM_USING_SSL value

File: OracleWebCenterPortal/dockerfiles/README.md [hunk 3]
```diff
@@ -45,18 +45,18 @@ $ docker tag container-registry.oracle.com/middleware/fmw-infrastructure:12.2.1.
...
+1. Clone or download the [Github repository](https://github.com/oracle/docker-images).The repository contains Docker files and scripts to build Docker images for Oracle products.
 
-The binaries can be downloaded from the [Here](https://www.oracle.com/middleware/technologies/webcenter-portal-download.html#).
+2. Download Oracle WebCenter Portal 12.2.1.4 binary from [Oracle Technology Network](https://www.oracle.com/middleware/technologies/webcenter-portal-download.html#).  
 
-Extract the downloaded zip files and copy `fmw_12.2.1.4.0_wcportal.jar` files under `dockerfiles/12.2.1.4` for building Oracle WebCenter Portal 12.2.1.4 image.
+3. Save the Oracle WebCenter Portal 12.2.1.4 binary into the cloned repository folder located at ../docker-images/OracleWebCenterPortal/dockerfiles/12.2.1.4/ and extract the downloaded zip.
```
Comment (dgothe) [inline OracleWebCenterPortal/dockerfiles/README.md:52]: instead of using ../docker-images, use $DIR/docker-images

**Comment by Djelibeybi** [comment]
You must rebase your fork and branches to the latest version from this repo. You're still carrying around commits from last year.

**Comment by Djelibeybi** [comment]
Also, please only assign me once the PR has been approved.

**Review by dgothe — APPROVED** [review]
Changes looks good

**Pull request merged**